src/index.js

```diff
@@ -12,9 +13,8 @@ export default function renderTasks() {
     const taskItem = document.createElement('li');
     taskItem.innerHTML += `<input class='checkbox' type="checkbox" ${task.completed ? 'checked' : ''}/><p>${task.description}</p> <div class="icons"> <i class="fa tri-dots">&#xf142;</i>
```

Comment:
* When I clear completed tasks and refresh my browser the deleted tasks comes back, according to the [requirement ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Store%20the%20updated%20array%20of%20items%20in%20local%20storage%2C%20so%20the%20user%20gets%20the%20correct%20list%20of%20values%20after%20the%20page%20reloads%20(which%20means%20that%20any%20changes%20made%20to%20the%20list%20should%20be%20preserved).)`Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads (which means that any changes made to the list should be preserved). `

https://user-images.githubusercontent.com/52497246/236382058-bef8bfcb-ce42-461a-a462-0d5faacfe490.mp4

Reply:
Well resolved ✔️ 

Great job preserving data after clearing completed tasks. 👏🏾 💯 

[OPTIONAL] I think it would be best if you also preserved data after `editing` a task. Currently, if I edit a task and then refresh the page, the modified task is restored back to its previous state.